### PR TITLE
Bug 1221307 - Step 1: Modularize routes

### PR DESCRIPTION
**Pull request by eliperelman**

### CODE REVIEW

File: src/device.js [hunk 1]
```diff
@@ -12,5 +12,5 @@ module.exports = async function device(req, res) {
   ]);
 
   let result = Object.assign({}, {gaia: gaiaCommit});
-  res.send(JSON.stringify(result));
+  res.json(result);
```
Comment (gaye) [inline src/device.js]: ooo nice

File: src/routes/crashes.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+          .filter(line => !/^\s*$/.test(line))
+          .map(line => line.slice(0, line.lastIndexOf('.')));
+      })
+    )
+  );
+
+  res.json(result);
+}
+
+router.get('/', listCrashReports);
```
Comment (gaye) [inline src/routes/crashes.js]: These are the statements that I'd like to keep in the top-level server file

Comment (eliperelman) [inline src/routes/crashes.js]: The router definitions are relative to the top-level route, which keeps them modular if you decide to refactor your routes. The top level route mapping to endpoints are still done at the services.js file, but the route controls its interior logic.

File: src/routes/device.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+let express = require('express');
```
Comment (gaye) [inline src/routes/device.js]: My one nit is that these things have gotten out of order!

Comment (gaye) [inline src/routes/device.js]: (Here and elsewhere : )

Comment (eliperelman) [inline src/routes/device.js]: LOL, no worries. :)

**Pull request merged**